### PR TITLE
Fix tab in sam_bitwise_flag_filter, and use print function

### PR DESCRIPTION
**Pull request by natefoo**
Fixes:

```pytb
  File "/cvmfs/main.galaxyproject.org/migrated_tools/toolshed.g2.bx.psu.edu/repos/devteam/sam_bitwise_flag_filter/0b2424a404d9/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py", line 111
    infile = open ( options.input_sam, 'r')
                                          ^
TabError: inconsistent use of tabs and spaces in indentation
```

I thought python 2 cared about this as well, so I'm not sure how this tool ever worked. Maybe it didn't?

### CODE REVIEW

File: tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py [hunk 1]
```diff
@@ -108,7 +109,7 @@ def main():
     options, args = parser.parse_args()
 
     if options.input_sam:
-		infile = open ( options.input_sam, 'r')
+        infile = open ( options.input_sam, 'r')
```
Comment (ic4f) [inline tools/sam_bitwise_flag_filter/sam_bitwise_flag_filter.py:112]:
```suggestion
        infile = open(options.input_sam, 'r')
```

**Comment by bernt-matthias** [comment]
I guess only the flake8 problems are left.

**Comment by bernt-matthias** [comment]
Hi @natefoo https://github.com/natefoo/tools-devteam/pull/1

`<paralellism>` seems to be broken (for containerized tests https://github.com/galaxyproject/galaxy/pull/9848)

**Comment by natefoo** [comment]
@bernt-matthias it looks like there's a merge issue on your PR?

**Comment by bernt-matthias** [comment]
Merged it manually.

**Comment by bernt-matthias** [comment]
Rebased https://github.com/natefoo/tools-devteam/pull/1 .. should be fine now

**Comment by bernt-matthias** [comment]
Closing in favor of https://github.com/galaxyproject/tools-devteam/pull/587

**Pull request closed without merging**